Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -94,10 +94,11 @@ to create a Kubernetes cluster locally or using a managed Kubernetes service:
 
        Install minikube >= v1.5.2 as per minikube documentation: 
        `Install Minikube <https://kubernetes.io/docs/tasks/tools/install-minikube/>`_.
+       The following command will bring up a single node minikube cluster configured to use the cni plugin. It will not have a cni configured.
```

Comment:
```suggestion
       The following command will bring up a single node minikube cluster prepared to get Cilium installed into.
```

Reply:
> It will not have a cni configured.

I think this bit is slightly misleading as `/etc/cni/net.d/087-podman-bridge.conflist` still gets installed, and hence pods can still run (e.g. CoreDNS starts, and any other pod you create will start as well) — there just won't be cross-node connectivity as far as I could tell. I propose we remove this bit, plus we do a slight rewording. 🙂

Reply:
it's true. This is a bug tho and should be corrected soon.

Reply:
comments addressed! Thanks!